src/adv.py

```diff
@@ -33,32 +34,149 @@
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
 
+# Adding some items in each room
+
+room['outside'].items.extend(
+    [Item("key", "Rusty looking key"), Item("torch", "9000 Lumens Torch")])
+room['foyer'].items.append(
+    Item("scrap", "scrap of paper, it has something written:Find the teasure"))
+room['overlook'].items.append(Item("binocular", "provides 100x zoom"))
+room['narrow'].items.append(
+    Item("ash", "pile of ash, probably someone set a fire"))
+room['treasure'].items.extend([Item("crown", "rusty looking crown"), Item(
+    "dagger", "it may deliver 5000x damage only if it had better handle")])
+
+
 #
 # Main
 #
 
 # Make a new player object that is currently in the 'outside' room.
 
-player = Player("Adnan", room['outside'].name)
+player = Player("Adnan", room['outside'])
 
 # Write a loop that:
 
 
-def adventure_game():
-    print("Welcome to the Adventure Game")
-    print(f"Welcome {player.name}!")
-    print(f"Your Current Room is {player.current_room}")
+def roomItems():
+    if len(player.current_room.items) > 0:
+        print('There are some items here:\n')
+        for item in player.current_room.items:
+            print(f"{item.getItemName()}")
 
```

Comment:
Nice! Creating sub functions to abstract code rather than long `if else` chains helps with readability a ton.